starknet_py/net/client_models.py

```diff
@@ -742,56 +634,6 @@ class TransactionStatusResponse:
     execution_status: Optional[TransactionExecutionStatus] = None
 
 
-@dataclass
-class SignatureInput:
-    """
-    Dataclass representing a signature input.
-    """
-
-    block_hash: int
-    state_diff_commitment: int
-
-
-@dataclass
-class SignatureOnStateDiff:
-    """
-    Dataclass representing signature on state diff commitment and block hash.
-    """
-
-    block_number: int
-    signature: List[int]
-    signature_input: SignatureInput
-
-
-class DAMode(Enum):
-    """
-    Enum specifying a storage domain in Starknet. Each domain has different gurantess regarding availability.
-    """
-
-    L1 = "L1"
-    L2 = "L2"
```

Comment:
Might need this for RPC 0.6.0, although it can probably be added back when needed 😅

Reply:
Good point! I am fine to remove it for now though as it not a problem to add it back later if necessary.